src/plays/pokemon-stats/components/Pokemoncard.tsx

```diff
@@ -1,6 +1,42 @@
 import React from "react";
 import "./pokecard.css";
-const Pokemoncard = (props) => {
+export interface Species {
+  name: string;
+}
+
+export interface Sprites {
+  front_default: string;
+}
+
+export interface Stat {
+  base_stat: number;
+  stat: Stat2;
+}
+
+export interface Stat2 {
+  name: string;
+}
+
+export interface Type {
+  type: Type2;
+}
+
```

Comment:
Can we create a separate file for interfaces?

Reply:
Sure i will .